etc/docker/demo/README.rst

```diff
@@ -0,0 +1,118 @@
+..  Copyright 2018 CERN for the benefit of the ATLAS collaboration.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+
+     Authors:
+   - Thomas Beermann <thomas.beermann@cern.ch>, 2017
+   - Cedric Serfon <cedric.serfon@cern.ch>, 2018
+   - Vincent Garonne <vgaronne@gmail.com>, 2018
+
+===================================
+Setting up a Rucio demo environment
+===================================
+
+Prerequisites
+--------------
+
+Setting up a Rucio demo environment requires to have `docker` and `docker-compose`
+installed. Docker is an application that makes it simple and easy to run
+application processes. To install Docker for your platform, please refer to
+the `Docker installation guide <https://docs.docker.com/install/>`_.
+
+The containers provided here can be used to easily setup a small demo instance of
+Rucio with some mock data to play around with some Rucio commands.
+
+docker-compose
+---------------
+
+A YAML file for `docker-compose` has been provided to allow easily setup of the containers.
+to have access to this `docker-compose.yml file <https://github.com/rucio/rucio/blob/master/etc/docker/demo/docker-compose.yml>`_,
+you can either:
+
+- `clone <https://help.github.com/articles/cloning-a-repository/>`_ the `Rucio repository <https://github.com/rucio/rucio/>`_,
+- `fork and clone <https://help.github.com/articles/fork-a-repo/>`_ the `Rucio repository <https://github.com/rucio/rucio/>`_ for the ones you want to `contribute <https://github.com/rucio/rucio/blob/master/CONTRIBUTING.rst>`_ to Rucio,
+- or download directly the file from the `Rucio repository <https://github.com/rucio/rucio/>`_, e.g,::
+
+    > $ wget https://raw.githubusercontent.com/rucio/rucio/master/etc/docker/demo/docker-compose.yml
```

Comment:
This will not work. The rucio container has to be build locally for the demo with the configuration files and certificates provided in the `etc/docker/demo` folder. So the build would fail here.

Reply:
ok so clone only ;)

Reply:
or wget the whole folder. Don't know if that's possible with github

Reply:
I added git in the prerequisites